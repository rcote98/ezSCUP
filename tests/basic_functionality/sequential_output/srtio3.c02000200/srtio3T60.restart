2	2	2
5	3
Sr	Ti	O
-3.00000000E-02	-3.00000000E-02	0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	1	1	1.78053960E-02	-1.21058299E-02	-1.15572356E-02
0	0	0	2	2	-7.67637415E-03	4.81073522E-04	-3.83816778E-04
0	0	0	3	3	1.60740195E-01	1.57129804E-01	1.45456745E-02
0	0	0	4	3	4.89095698E-01	-1.85084340E-03	-1.68263533E-01
0	0	0	5	3	-3.58293289E-03	-5.09920631E-01	-1.82284327E-01
0	0	1	1	1	-1.35819909E-02	-4.37147448E-03	4.53896856E-03
0	0	1	2	2	8.65593384E-03	1.58115889E-04	-6.80657184E-04
0	0	1	3	3	-2.00123477E-01	-1.30779987E-01	-1.07452190E-02
0	0	1	4	3	-4.76448051E-01	-1.37648378E-03	1.62407921E-01
0	0	1	5	3	6.57402499E-03	5.00288441E-01	1.70833714E-01
0	1	0	1	1	-1.57333371E-02	-6.49408956E-03	1.07426321E-02
0	1	0	2	2	8.84473499E-03	7.26905579E-03	9.38383344E-03
0	1	0	3	3	-1.49418358E-01	-1.36266565E-01	1.32025217E-02
0	1	0	4	3	-5.11994039E-01	6.73304026E-03	1.47824924E-01
0	1	0	5	3	-2.68781182E-03	4.78614496E-01	1.49997832E-01
0	1	1	1	1	-1.05589642E-02	3.23051734E-02	1.54130850E-02
0	1	1	2	2	8.85166723E-03	-1.85932689E-02	2.41540307E-02
0	1	1	3	3	1.44202726E-01	1.73584502E-01	1.22396951E-03
0	1	1	4	3	4.84063174E-01	-3.23558129E-03	-1.77147173E-01
0	1	1	5	3	-4.92739144E-04	-4.71314071E-01	-1.28555165E-01
1	0	0	1	1	1.07226058E-02	3.40360270E-03	-1.01901652E-02
1	0	0	2	2	-3.24634016E-03	-1.24783019E-02	-3.08434264E-02
1	0	0	3	3	-1.93445996E-01	-1.76882893E-01	-6.18647240E-03
1	0	0	4	3	-4.80269743E-01	-1.61596109E-02	1.66804648E-01
1	0	0	5	3	8.42608322E-04	4.84157861E-01	1.68562358E-01
1	0	1	1	1	1.55340523E-02	-1.58342484E-03	-1.87413351E-02
1	0	1	2	2	2.30676027E-03	-4.44501723E-03	-3.35249663E-02
1	0	1	3	3	1.88465834E-01	1.64440324E-01	-3.07371740E-03
1	0	1	4	3	4.90495663E-01	-4.92753137E-03	-2.20404653E-01
1	0	1	5	3	-1.18737450E-03	-5.01639162E-01	-1.87915198E-01
1	1	0	1	1	2.55455101E-02	2.22210670E-02	2.14593006E-02
1	1	0	2	2	6.56613943E-03	-9.52713624E-03	2.74713298E-02
1	1	0	3	3	1.37840017E-01	1.85661293E-01	1.59890799E-02
1	1	0	4	3	4.89252552E-01	-1.54152859E-02	-1.72235577E-01
1	1	0	5	3	1.69615007E-03	-4.81834801E-01	-1.19793542E-01
1	1	1	1	1	-1.02306861E-02	-1.08516527E-02	-3.31114382E-02
1	1	1	2	2	7.93657402E-03	4.95154909E-03	2.84306214E-02
1	1	1	3	3	-1.52524641E-01	-1.89364449E-01	1.29557814E-02
1	1	1	4	3	-4.83964296E-01	6.88531617E-03	1.64995760E-01
1	1	1	5	3	1.11291362E-02	4.93133377E-01	1.84699637E-01
