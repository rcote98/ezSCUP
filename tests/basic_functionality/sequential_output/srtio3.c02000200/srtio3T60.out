
*****************************************************************
*                          Welcome to                           *
*                           SCALE-UP                            *
*                                                               *
*            Second-Principles Computational Approach           *
*                             for                               *
*                     Lattice and Electrons                     *
*                                                               *
*                       Version:         1.0.0.Gorbea           *
*                    Build date: 20200801T181                   *
*****************************************************************
*                    SCALE-UP is property of                    *
*              the University of Cantabria (Spain)              *
*    and the Luxembourg Institute of Science and Technology     *
*****************************************************************
* Authors:                                                      *
*                                                               *
*  - Pablo Garcia-Fernandez   garciapa@unican.es                *
*  - Jorge Iñiguez            jorge.iniguez@list.lu             *
*  - Javier Junquera          junqueraj@unican.es               *
*****************************************************************
* SCALE-UP is based on two earlier programs:                    *
*                                                               *
* -  SPLD ('Second-Principles Lattice Dynamics'), developed     *
*    at the ICMAB-CSIC (Spain) by J.C. Wojdel, M.P. Ljungberg   *
*    and J. Iñiguez, and distributed under MIT license.         *
*                                                               *
* -  MKII a second-principles electron code                     *
*    developed at the University of Cantabria by                *
*    P. Garcia-Fernandez  and J. Junquera.                      *
*****************************************************************
*                                                               *
* Bibliography:                                                 *
*                                                               *
*  P. Garcia-Fernandez et al. Phys. Rev. B 93, 195137 (2016)    *
*  J. C. Wojdel et al., J. Phys. Cond. Matt. 25, 305401 (2013)  *
*****************************************************************
Reading parameter file... lattice... done!
Reading restart file     srtio3T60.restart                                                               

Basic data:
  Number of CPUs:               1
  Run mode:            Montecarlo
  Lattice:                 Active
  Electrons:           NOT Active
  External electric field, static. Field components (V/m):   0.000000E+00   0.000000E+00   0.000000E+00

Reference system:
 Supercell data:
  Lattice constants (Ang):   7.690   7.690   7.690
  Lattice angles (degree):  90.000  90.000  90.000
 Supercell size:   2   2   2

 Total number of atoms          :      40
 Total number of orbitals       :       0
 Total number of active orbitals:       0

 Atom in unit cell:
 Atom   Label       Mass  Charge 
    1   Sr         87.60     0.0
    2   Ti         47.90     0.0
    3   O          16.00     0.0
    4   O          16.00     0.0
    5   O          16.00     0.0

 Orbitals in unit cell:
 Orbital   Label  

 Background dielectric tensor:
    6.2319     0.0000     0.0000
    0.0000     6.2319    -0.0000
    0.0000    -0.0000     6.2319

Initial values (basic output MC simulation)
Energy (eV):            -0.298477E+05
Polarization (C/m2):     0.006862   -0.001764   -0.000268
Strain:                 -0.030000   -0.030000    0.000000    0.000000    0.000000    0.000000

LT:    Iter       Temp(K)    Energy(eV)   Pol_x(C/m2)   Pol_y(C/m2)   Pol_z(C/m2) Av_Pl_x(C/m2) Av_Pl_y(C/m2) Av_Pl_z(C/m2)       Strn_xx       Strn_yy       Strn_zz       Strn_yz       Strn_xz       Strn_xy    Av_Strn_xx    Av_Strn_yy    Av_Strn_zz    Av_Strn_yz    Av_Strn_xz    Av_Strn_xy 
LT:      10  0.600000E+02 -0.298475E+05 -0.269821E-01  0.207649E-01 -0.249465E-01 -0.576921E-02  0.106398E-01  0.202864E-02 -0.300000E-01 -0.300000E-01  0.446466E-03  0.141559E-03  0.257294E-02  0.000000E+00 -0.330000E-01 -0.330000E-01 -0.111268E-03  0.869805E-03  0.252364E-03  0.000000E+00 
LT:      20  0.600000E+02 -0.298475E+05 -0.157569E-01  0.302039E-01  0.139280E-01 -0.613175E-02  0.177071E-01 -0.479518E-02 -0.300000E-01 -0.300000E-01  0.202363E-02 -0.131370E-02  0.143194E-02  0.000000E+00 -0.315000E-01 -0.315000E-01  0.752076E-03  0.366965E-03  0.149248E-02  0.000000E+00 
LT:      30  0.600000E+02 -0.298476E+05  0.206115E-01  0.366912E-02  0.324744E-01 -0.264610E-02  0.191275E-01  0.242435E-02 -0.300000E-01 -0.300000E-01  0.319562E-02  0.106498E-02  0.868370E-04  0.000000E+00 -0.310000E-01 -0.310000E-01  0.131969E-02  0.100825E-03  0.781911E-03  0.000000E+00 
LT:      40  0.600000E+02 -0.298476E+05  0.652611E-03 -0.512553E-01  0.395672E-01  0.171844E-02  0.119921E-01  0.338706E-02 -0.300000E-01 -0.300000E-01  0.397181E-02 -0.114052E-02  0.142009E-02  0.000000E+00 -0.307500E-01 -0.307500E-01  0.205437E-02  0.997371E-04  0.114030E-02  0.000000E+00 
LT:      50  0.600000E+02 -0.298478E+05  0.995588E-02 -0.187585E-02  0.195509E-02  0.265912E-02  0.275002E-02  0.622540E-02 -0.300000E-01 -0.300000E-01  0.946544E-02 -0.522463E-02  0.311194E-02  0.000000E+00 -0.306000E-01 -0.306000E-01  0.311082E-02 -0.637104E-03  0.109040E-02  0.000000E+00 
LT:      60  0.600000E+02 -0.298477E+05  0.212192E-01  0.183190E-01  0.472569E-03  0.261873E-02  0.637413E-02  0.581896E-02 -0.300000E-01 -0.300000E-01  0.130390E-01 -0.522941E-02  0.250160E-02  0.000000E+00 -0.305000E-01 -0.305000E-01  0.448509E-02 -0.138382E-02  0.137167E-02  0.000000E+00 
LT:      70  0.600000E+02 -0.298478E+05  0.199085E-01 -0.202367E-01  0.548320E-01  0.397204E-02  0.779279E-02  0.638747E-02 -0.300000E-01 -0.300000E-01  0.198840E-01 -0.680929E-02  0.251567E-02  0.000000E+00 -0.304286E-01 -0.304286E-01  0.635658E-02 -0.232146E-02  0.174670E-02  0.000000E+00 
LT:      80  0.600000E+02 -0.298478E+05 -0.177813E-01 -0.275154E-01  0.172994E-03  0.508930E-02  0.300582E-02  0.972695E-02 -0.300000E-01 -0.300000E-01  0.246437E-01 -0.683247E-02 -0.129617E-03  0.000000E+00 -0.303750E-01 -0.303750E-01  0.855192E-02 -0.255999E-02  0.153595E-02  0.000000E+00 
LT:      90  0.600000E+02 -0.298478E+05 -0.147126E-01  0.837530E-03  0.216938E-01  0.284724E-02  0.110141E-02  0.764916E-02 -0.300000E-01 -0.300000E-01  0.243556E-01 -0.395407E-02 -0.114487E-02  0.000000E+00 -0.303333E-01 -0.303333E-01  0.102335E-01 -0.278845E-02  0.115595E-02  0.000000E+00 
LT:     100  0.600000E+02 -0.298478E+05 -0.305096E-02  0.697011E-02 -0.634373E-02  0.236892E-02  0.233150E-02  0.862166E-02 -0.300000E-01 -0.300000E-01  0.191549E-01  0.221085E-02 -0.115071E-02  0.000000E+00 -0.303000E-01 -0.303000E-01  0.115575E-01 -0.249558E-02  0.101883E-02  0.000000E+00 

Final values (basic output MC simulation)
Energy (eV):            -0.298478E+05
Polarization (C/m2):    -0.003051    0.006970   -0.006344
Strain:                 -0.030000   -0.030000    0.019155    0.002211   -0.001151    0.000000

-----------------------------------------------------------------
 Backing up the input read by fdf:

System_name     srtio3T60
Print_routineinfo         5     # default value
Print_routinedepth         0     # default value
Print_inout         0     # default value
Print_interactions         0     # default value
Print_k-vectors         0     # default value
Print_energies         1     # default value
Print_wavefunctions        -1     # default value
Print_electrostatic         1     # default value
Print_orbocc         1     # default value
Print_bands        -1     # default value
Print_forces        -1     # default value
Print_detail        -1     # default value
Print_Just_Geometry        -1     # default value
Print_Velocity         0     # default value
Print_Space_Hamiltonian        -1     # default value
Print_space_denmat        -1     # default value
Print_Geometry         1     # default value
Print_geoformat     normal     # default value
Print_test_file        -1     # default value
Print_local_lattice_energy        -1     # default value
Print_local_current        -1     # default value
Print_std_electron_nsteps         1     # default value
Print_std_lattice_nsteps             10
Print_std_energy              T
Print_std_polarization              T
Print_std_current              F     # default value
Print_std_strain              T
Print_std_av_polarization              T
Print_std_av_strain              T
Print_std_av_energy              F     # default value
Print_std_delta_energy              F     # default value
Print_std_temperature              T     # default value
Print_std_maximum_force              F     # default value
Print_std_maximum_stress              F     # default value
Print_std_mdd              T     # default value
Print_std_fermi              T     # default value
Print_std_pulay              T     # default value
Print_std_neh              T     # default value
Print_std_el_pol              T     # default value
Print_std_micro              F     # default value
run_mode     monte_carlo
Print_std_mdd              F     # default value
Print_std_fermi              F     # default value
Print_std_pulay              F     # default value
Print_std_neh              F     # default value
Print_std_electron_nsteps     10000     # default value
No_electron
%block fix_strain_component
 T T F F F T
T                                                                                             T
T                                                                                             T
F                                                                                             F
F                                                                                             F
F                                                                                             F
T                                                                                             T
Parameter_file     srtio3_full_lat.xml
%block Supercell
 2 2 2
%endblock Supercell
electric_field_scale          1.000000000     v/m     # default value
%block static_electric_field
 0.0 0.0 0.0
temperature         0.1000000000E-98 kelvin     # default value
%block external_stress
 0.0 0.0 0.0 0.0 0.0 0.0
Charge_tolerance         0.1000000000E-07     # default value
Geometry_restart
Geometry_restart     srtio3T60.restart
MC_nsweeps
MC_nsweeps            100
MC_temperature
MC_temperature          60.00000000         kelvin
# above item originally: MC_temperature          60.00000000     kelvin
MC_annealing_rate          1.000000000
MC_init_step_d         0.9448629925E-01 bohr     # default value
MC_init_step_s         0.1889725985E-02 bohr     # default value
MC_max_step_d         0.9448634389           bohr
# above item originally: MC_max_step_d         0.5000000000     ang
MC_min_step_d         0.1889725985E-05 bohr     # default value
MC_target_acc_d         0.5000000000         # default value
MC_target_acc_s         0.5000000000         # default value
MC_strains              T
n_write_mc
n_write_mc             10
Print_std_lattice_nsteps
*****************************************************************
 Memory usage:
  Function             Total_memory(Mb)  Max_memory(Mb)      Node
  setup_main                     0.0256          0.0256         0
  setup_basicvariables           0.0056          0.0056         0
  setup_latt_hamil               0.0014          0.0014         0
  setup_init_data_node           0.0256          0.0256         0
  hel_setup_structfactors        0.0256          0.0256         0
  hlt_create_supercell           0.0014          0.0014         0

 Timing statistics (seconds)
  Function                       Wall   Max_Wall    Max_CPU  Node
  setup_main                     3.76       3.76       3.73     0
  setup_readinput                3.54       3.54       3.51     0
  montecarlo_main                0.21       0.21       0.21     0
  montecarlo_sweep               0.21       0.21       0.20     0
  io_readparameterfile           3.54       3.54       3.51     0
  setup_latt_hamil               0.22       0.22       0.22     0
  hlt_add_harmonic_didi          0.22       0.22       0.22     0

 Total memory usage:     0.03 MB
 Total WALL:           3.977 seconds
*****************************************************************
*                                                               *
*               Succesfully ending SCALE-UP                     *
*                                                               *
*****************************************************************
